client.js

```diff
@@ -18,29 +18,93 @@ function createClientConfig() {
     }
 }
 
-(async () => {
-    try {
-        const client = await Client.newHazelcastClient(createClientConfig());
-        const map = await client.getMap('map');
-        await map.put("key", "value")
-        const res = await map.get("key")
-        if (res !== "value") {
-            throw new Error("Connection failed, check your configuration.")
-        }
-        console.log("Connection Successful!");
+function mapExample(map) {
+    (async () => {
         console.log("Now, `map` will be filled with random entries.");
 
         var iterationCounter = 0;
         while (true) {
             const randomKey = Math.floor(Math.random() * 100000);
-            await map.put('key' + randomKey, 'value' + randomKey)
+            await map.put('key' + randomKey, 'value' + randomKey);
             await map.get('key' + randomKey);
             if (++iterationCounter === 10) {
                 iterationCounter = 0;
                 map.size().then((size) => console.log(`map size: ${size}`));
             }
         }
+    })();
+}
+
+function sqlExample(hzClient) {
+    (async () => {
+        console.log("Creating a mapping...");
+        // See: https://docs.hazelcast.com/hazelcast/5.0/sql/mapping-to-maps
+        await hzClient.getSql().execute("DROP MAPPING cities");
+        const mappingQuery = "CREATE MAPPING cities TYPE IMap OPTIONS " +
```

Comment:
instead of drop - create we can use "create or replace mapping"

Reply:
Agree with @ivan-shevtsov

Reply:
Good catch. Thanks, fixed.